src/v/redpanda/admin/server.cc

```diff
@@ -1257,8 +1257,8 @@ void admin_server::register_config_routes() {
               writer.String(name);
               if (include_levels) {
                   writer.Key("level");
-                  writer.String(fmt::to_string(
-                    ss::global_logger_registry().get_logger_level(name)));
+                  writer.String(fmt::format(
```

Comment:
What was the error?

Reply:
```
/home/robert/workspace/redpanda/vbuild/debug/clang/rp_deps_install/include/fmt/format.h:2183:41: error: non-const lvalue
 reference to type 'basic_format_context<detail::buffer_appender<char>, [...]>' cannot bind to a value of unrelated type
 'basic_format_context<std::back_insert_iterator<std::string>, [...]>'                                                  
  return formatter_type().format(value, ctx);                                                                           
                                        ^~~                                                                             
/home/robert/workspace/redpanda/vbuild/debug/clang/rp_deps_install/include/fmt/format.h:2884:11: note: in instantiation 
of function template specialization 'fmt::v8::detail::write<char, std::back_insert_iterator<std::string>, seastar::log_l
evel, fmt::basic_format_context<std::back_insert_iterator<std::string>, char>>' requested here                          
  detail::write<char>(std::back_inserter(result), value);                                                               
          ^                                                                                                             
/home/robert/workspace/redpanda/src/v/redpanda/admin/server.cc:1261:38: note: in instantiation of function template spec
ialization 'fmt::to_string<seastar::log_level, 0>' requested here                                                       
                  writer.String(fmt::to_string(                                                                         
                                     ^                                                                                  
/home/robert/workspace/redpanda/vbuild/debug/clang/rp_deps_install/include/seastar/util/log.hh:72:64: note: passing argu
ment to parameter 'ctx' here                                                                                            
    auto format(seastar::log_level level, fmt::format_context& ctx) const -> decltype(ctx.out());                       
```

Reply:
Link to issue to resolve later: https://redpandadata.atlassian.net/jira/software/c/projects/CORE/issues/CORE-2755